landing-page/scripts/inference_api.py

```
@@ -102,4 +106,128 @@ async def get_staffline_detection(
+    if not ann_row:
+        raise HTTPException(status_code=404, detail=f"no current annotation for image {image_id}")
+    annotation_id, yolo_txt = ann_row
+
+    return image_id, image_name, annotation_id, image_arr, yolo_txt
+
+
+@router.post("/projects/{project_id}/stafflines/{detection_id}/interpolate-preview")
+async def preview_staffline_interpolation(
+    project_id: int,
+    detection_id: str,
+    user=Depends(get_current_user),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)